Documentation/running-clair.md

```diff
@@ -83,6 +56,11 @@ $ docker run --net=host -d -p 6060-6061:6060-6061 -v $PWD/clair_config:/config q
 
 #### Source
 
+Clair currently uses glide as it's package management solution. The vendor folder is checked into this repository and running 'glide up' should not be necessary.
+Find more information on glide here: [Glide]
+
+[Glide]: https://github.com/Masterminds/glide
+
```

Comment:
How do you feel about removing the source section in general?
It seems semi-redundant with the local development doc now.

Reply:
@jzelinskie there was a bit of talk here: https://github.com/coreos/clair/issues/790 about having "general" running of clair information. IMO it does not hurt to have source build info if ppl want to just build the binary in their CI systems or other needs.

Reply:
also not sure where else we'd mention the required binaries in path?

Reply:
ping @jzelinskie

Reply:
Forget it then, this is fine for now.